docs/explanations/packaging-data.md

```diff
@@ -0,0 +1,158 @@
+
+Some manufacturers send us more detailed packaging data (e.g. recycling instructions) through the Producers Platform.
+
+Some manufacturers send us data used to compute the Eco-Score using the Eco-Score spreadsheet template, which has fields like "Packaging 1", "Material 1", "Packaging 2", "Material 2" etc.
+
+### Product photos and machine learning
+
+We can extract logos related to packaging, or parse the text recognized from product photos to recognize packaging information or recycling instructions.
+
+## How packaging data is currently added, updated and structured in Open Food Facts
+
+In Open Food Facts, we currently have a number of input fields related to packaging. The data in those fields is parsed and analyzed to create a structured list of packaging components with specific attributes.
+
+### Current input fields
+
+#### Packaging tag field (READ and WRITE)
+
+At the start of Open Food Facts in 2012, we had a "packaging" tag field where users could enter comma separated free text entries about the packaging (e.g. "Plastic", "Bag" or "Plastic bag") in different languages.
+
+In 2020, we made this field a taxonomized field. As a result, we now store the language used to fill this field, so that we can match its value to the multilingual packaging taxonomy. So "plastique" in French will be mapped to the canonical "en:plastic" entry.
+
+#### Packaging information / recycling instructions text field (READ and WRITE)
+
+In 2020, we also added a language specific field ("packaging_text_[language code]" e.g. "packaging_text_en" for English) to store free text data about the packaging. It can contain the text of the recycling instructions printed on the packaging (e.g. "bottle to recycle, cap to discard"), or can be filled in by users (e.g. "1 PET plastic bottle to recycle, 1 plastic cap").
+
+### Current resulting packagings data structure (READ only)
+
+The input fields are analyzed and combined to create the "packagings" data structure.
+
+The structure is an array of packaging components. Each packaging component can have values for different attributes:
+
+- number: the number of units for the packaging component (e.g. a pack of beers may contain 6 bottles)
+- shape: the general shape of the packaging component (e.g. "bottle", "box")
+- material: the material of the packaging component
+- quantity: how much product the packaging component contains (e.g. "25 cl")
+- recycling: whether the packaging component should be recycled, discarded or reused
+
+The "shape" and "material" fields are taxonomized using the packaging_shapes and packaging_materials taxonomies.
+
+### How the the resulting packagings data structure is created
+
+The values for each input field ("packaging" tag field and "packaging_text_[language code]" packaging information text field) are analyzed to recognize packaging components and their attributes.
```

Comment:
You don't explain how we deal with entries in more than one language. Do we analyze them all ?

Reply:
Currently we only analyze the entry corresponding to the main language of the product. Added a note about that.